test/jdk/javax/swing/JTableHeader/JTableHeaderLabelRightAlignTest.java

```diff
@@ -88,15 +88,18 @@ private static void test() {
 
         for (int y = 1; y < (imgHeader.getHeight() - 3); y++) {
             if (expectedRGB != imgHeader.getRGB(x, y)) {
-                try {
-                    ImageIO.write(imgHeader, "png",
-                            new File("FailureImage.png"));
-                } catch (IOException ioException) {
-                    ioException.printStackTrace();
-                }
+                saveBufferedImage(imgHeader, "failureImage.png");
                 throw new RuntimeException("Test Failed at <" + x + ", " + y + ">");
             }
         }
         System.out.println("Test Passed");
     }
+
+    private static void saveBufferedImage(BufferedImage image, String fileName) {
```

Comment:
`saveImage` should be enough as in #14766?

Reply:
I have used `saveBufferedImage` in both the test/PR, I think it should be fine.

Reply:
My point is that `saveImage` clearly conveys the meaning, you pass a `BufferedImage`, which implies you save a buffered image. Whatever, but shorter name is better in my opinion, and it's used in many other tests.